package.json

```diff
@@ -41,12 +42,9 @@
     "unused": true
   },
   "dependencies": {
+    "google-closure-library": "^20171203.0.0",
     "install": "^0.8.8",
     "npm": "^4.4.4",
-    "closure-library": "^1.43629075.2",
```

Comment:
This is an abandoned unofficial 5 year old copy of Closure Library, not the real one.
https://github.com/gregrperkins/closure-library